pkg/reconciler/v1alpha1/pipelinerun/pipelinerun_test.go

```diff
@@ -479,10 +502,15 @@ func TestReconcileOnCompletedPipelineRun(t *testing.T) {
 	c.Reconciler.Reconcile(context.Background(), "foo/test-pipeline-run-completed")
 
 	// make sure there is no failed events
-	validateEvents(t, fr)
+	validateNoEvents(t, fr)
```

Comment:
i think this is b/c we don't have the success event anymore? (just not clear to me if this was an intentional change or not!)

Reply:
Yup, good catch - I was wondering how it was passing before.

Reply:
Oh actually, now that I think about it - I think I removed that event because it was emitted for both pipeline success and failure. Does that make sense?

Reply:
@dicarlo2 can you explain a bit more? I think we'd want to have an event in both cases - i.e. one event in each case, just to try to be very clear:

- Failure: event indicating failure
- Success: event indication success

(maybe that is already happening?)

Reply:
Sorry, to be clear, I removed it because we seemed to be emitting it in error, or at least it didn't make sense because regardless of success/failure we were always emitting a success event. Looking at it again, I think that would be the wrong place to emit an event anyways since we would essentially emit an event every time the controller resynchronizes and runs reconciliation on a completed/stale `PipelineRun`.

Also, it appears we're already emitting a success/failure event here: https://github.com/tektoncd/pipeline/blob/0b42a421ab5c4024d3bdd4ee7ecd118e55a3b384/pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go#L374-L379

(and emitting it appropriately, i.e. it should be emitted only once on final success/error status change)